cli/delete.go

```diff
@@ -44,6 +52,15 @@ func deleteProjectCommand(t *core.Timetrace) *cobra.Command {
 			key := args[0]
 
 			if options.Revert {
+				if !options.ExcludeRecords && askForConfirmation(revertRecordsWarning) {
```

Comment:
Note: We may outsource this entire block into a dedicated `revertProject` function later.